test/checkpoint/test_checkpoint.c

```diff
@@ -333,7 +336,7 @@ static int
 usage(void)
 {
     fprintf(stderr,
-      "usage: %s [-C wiredtiger-config] [-c checkpoint] [-h home] [-k keys]\n\t[-l log] [-n ops] "
+      "usage: %s [-C wiredtiger-config] [-c checkpoint] [-h home] [-k keys]\n\t[-l log] [-n ops]"
       "[-r runs] [-T table-config] [-t f|r|v]\n\t[-W workers]\n",
```

Comment:
Should probably update this part with `|p`.

Reply:
I was planning to do that but I think we only options with parameters here. `f|r|v` are table types not options.

Reply:
Ah, I see.